rca/static_src/sass/components/_page.scss

```diff
@@ -120,23 +128,6 @@
         }
     }
 
-    .sticky-bar.app--short-course & {
```

Comment:
As far as I can tell, this doesn't do anything. Whitespace is already created via padding on the block wrappers, and with this in place text is obscured on mobile. E.g [on this page](https://www.rca.ac.uk/study/programme-finder/service-design-masterclass/), the text `Use the principles and tools of service design to rethink the journey your customers take and transform your customer experience, delivered live online.` is hidden.

Removing it didn't appear to harm anything, but it'd be good to double check with someone who has more context of the project.

Reply:
My understanding was that the sticky book bar used to sit at the top of the page, so when the mini header came in, it had to move down.

It appears to have since been fixed to the bottom, so this is no longer required.